cneuromax/projects/friends_language_encoder/peftmodule.py

```diff
@@ -21,21 +20,25 @@ class PEFTLitModule(BaseLitModule, ABC):
 
     def __init__(
         self: "PEFTLitModule",
-        config: BaseLitModuleConfig,
-        nnmodule: nn.Module,
-        optimizer: partial[Optimizer],
-        scheduler: partial[LRScheduler],
+        config: "BaseLitModuleConfig",
+        peft_config: "PeftConfig",
```

Comment:
doesn't need to be in parenthesis

Reply:
do you mean like this? 
```
@dataclass
class PEFTLitModule(BaseLitModule, ABC):
    """``project`` :class:`~PEFTLitModule`."""

    def __init__(
        self: "PEFTLitModule",
        *args: Any,  # noqa: ANN401, ARG002
        **kwargs: Any,  # noqa: ANN401, ARG002
    ) -> None:
        """."""
        super().__init__(
            *args,
            **kwargs,
        )
        peft_config = PeftConfig()
        self.nnmodule = get_peft_model(self.nnmodule, peft_config)
```

Reply:
my bad quotations not parenthesis, meant that `peft_config: "PeftConfig"` can turn to `peft_config: PeftConfig`

Reply:
ah ok makes sense, otherwise it complains for sure!